.vscode/launch.json

```diff
@@ -46,6 +45,12 @@
       "outFiles": [
         "${workspaceFolder}/**/*.js"
       ]
+    },
+    {
+      "type": "node",
+      "request": "launch",
+      "name": "Launch Integrations Example",
+      "program": "${workspaceFolder}/examples/integrations/index.js"
```

Comment:
I will suggest to rename this example from `integrations` to `rest` or someting like that. 
Integrations are too generic thing